extractor/controller.scala

```diff
@@ -0,0 +1,105 @@
+package morpheus
+package extractors
+
+import scala.meta._
+import scala.meta.dialects.Scala211
+
+package object controller {
+
+  private[this] def extractMethod(m: Decl.Def): String =
+    m.mods.collectFirst {
+      case Mod.Annot(Ctor.Ref.Name("query")) => "get"
+      case Mod.Annot(Ctor.Ref.Name("command")) => "post"
+    }.get
```

Comment:
`get` is safe here, because we explicitly filtered only the methods that have either annotation.